### PR TITLE
Refactor for new buttons

### PR DESCRIPTION
**Pull request by ingedmundo**
@ijnebchavez did some changes on the buttons design so I had to adjust styles

image

### CODE REVIEW

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -738,18 +749,27 @@ blockquote {
...
+    background: transparent;
+    border-color: #EEEEEE;
+    color: #DDDDDD; }
 
 .btn-3 {
-  background-color: #DDDDDD;
-  border: 5px solid #DDDDDD;
-  color: #FFFFFF;
+  background-color: transparent;
+  border-color: #DDDDDD;
```
Comment (vicmaster) [inline public/stylesheets/main.css]: Try to use the group attributes...for example: border: 4px solid #DDD;

File: public/stylesheets/main.css [hunk 2]
```diff
@@ -705,11 +705,13 @@ blockquote {
 
 .btn-1 {
   background-color: transparent;
-  border: 5px solid #DDDDDD;
-  color: #CCCCCC;
+  border-color: #FFFFFF;
```
Comment (vicmaster) [inline public/stylesheets/main.css]: Try to use the group attributes...for example: border: 4px solid #FFF;

File: assets/stylesheets/mixins.scss [hunk 3]
```diff
@@ -5,13 +8,18 @@
...
 }
 
-@mixin buttons($bg-color, $border, $color, $hover-bg-color, $hover-color) {
+@mixin buttons($bg-color, $border-color, $color,
+                $hover-bg-color, $hover-color,
+                $active-bg-color, $active-color,
+                $disabled-border-color, $disabled-color) {
   background-color: $bg-color;
-  border: $border;
+  border-color: $border-color;
```
Comment (vicmaster) [inline assets/stylesheets/mixins.scss]: Try to use the group attributes...for example: border: $btn-border solid $border-color;

**Comment by rodacato** [comment]
LGTM

File: assets/stylesheets/mixins.scss [hunk 4]
```diff
@@ -1,3 +1,6 @@
+$btn-border: 4px;
+$disabled-lighten: 8%;
```
Comment (mumoc) [inline assets/stylesheets/mixins.scss]: I would prefer to have variables in a variable css, which is loaded before loading the mixins one... what do you think?

File: assets/stylesheets/mixins.scss [hunk 5]
```diff
@@ -5,13 +8,16 @@
   width: $width;
 }
 
-@mixin buttons($bg-color, $border, $color, $hover-bg-color, $hover-color) {
+@mixin buttons($bg-color, $border-color, $color,
```
Comment (mumoc) [inline assets/stylesheets/mixins.scss]: This is pretty weird ... not sure why would you ever need to send all those values but, couldn't you implement this using ```@mixin buttons($args...)``` ??

File: assets/stylesheets/mixins.scss (hunk 4 above)
Comment (ingedmundo) [inline assets/stylesheets/mixins.scss]: ... thanks, I noticed I dont need that variable anymore.

File: assets/stylesheets/mixins.scss (hunk 5 above)
Comment (mumoc) [inline assets/stylesheets/mixins.scss]: If you think about it, the method could be refactored, on ruby, maybe you would do something like:
```
def my_method (args)
  options = defaults.merge arges
 
  set_options(options)
end
```

why not trying to do something alike here? http://sass-lang.com/documentation/file.SASS_REFERENCE.html#maps

Comment (ingedmundo) [inline assets/stylesheets/mixins.scss]: What do you mean?  sending a single parameter with all the others inside?

File: assets/stylesheets/mixins.scss [hunk 6]
```diff
@@ -5,13 +8,16 @@
...
+                $active-bg-color, $active-color,
+                $disabled-border-color, $disabled-color) {
   background-color: $bg-color;
-  border: $border;
+  border: $btn-border solid $border-color;
   color: $color;
   cursor: pointer;
   display: inline-block;
-  font: normal 100 0.938em 'Lato', sans-serif;
+  font: 100 18px Lato;
```
Comment (mumoc) [inline assets/stylesheets/mixins.scss]: not sure if you should be using the ```rem``` function to set the font size

File: buttons.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <title>Crowd Interactive branding</title>
+    <link rel='stylesheet' href='public/stylesheets/main.css'>
+  </head>
+  <body>
+    <h1>Buttons</h1>
```
Comment (mumoc) [inline buttons.html]: not sure, but I does seems like you're not following your own html conventions ... aren't you missing something? amm like a section ? if I'm wrong just ignore my comment xD

**Comment by vicmaster** [comment]
Beside my comments, and further comments.. LGTM :)

**Pull request merged**